Add yarn install into bin/setup when not using importmap (#44717)
rails/rails: railties/CHANGELOG.md

    *Harun Sabljaković*

> *   Add JavaScript dependencies installation on bin/setup

Reviewer: New changelogs should be at the top of the file.

>     The bin/setup script now includes a `yarn install` step that will be executed when the project is generated using esbuild, webpack or rollup.

Reviewer: ```suggestion
    Add  `yarn install` to bin/setup when using esbuild, webpack, or rollout.
```
